src/components/interaction-designer/block-types/Core_LogBlock.vue

```diff
@@ -76,20 +73,23 @@ import FirstBlockEditorButton from '../flow-editors/FirstBlockEditorButton.vue'
 import BlockId from '../block-editors/BlockId.vue'
 import GenericContactPropertyEditor from '../block-editors/GenericContactPropertyEditor.vue'
 
-const flowVuexNamespace = namespace('flow')
+const blockVuexNamespace = namespace(`flow/${BLOCK_TYPE}`)
 const builderVuexNamespace = namespace('builder')
 
 @Component({
   components: {
     GenericContactPropertyEditor,
+    ExpressionInput,
     ResourceEditor,
     ResourceVariantTextEditor,
```

Comment:
Hi @jcbashdown just realized, we could also probably remove those two resource editor components 🤩

Reply:
Removed these and a couple of other extraneous components too.